_includes/head.html

```diff
@@ -0,0 +1,45 @@
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  {% if page.layout == "eip" %}
+    <title>EIP {{ page.eip }}: {{ page.title | escape }}</title>
+    <meta property="og:title" content="EIP {{ page.eip }}: {{ page.title | escape }}" />
+    <meta
+      name="description"
+      content="Details on Ethereum Improvement Proposal {{page.eip}} (EIP {{page.eip}}): {{page.title | escape}}"
+    />
+    <meta
+      property="og:description"
+      content="Details on Ethereum Improvement Proposal {{page.eip}} (EIP {{page.eip}}): {{page.title | escape}}"
+    />
+    <meta name="twitter:description" content="Details on Ethereum Improvement Proposal {{page.eip}} (EIP {{page.eip}}): {{page.title | escape}}" />
+  {% else %}
+  <title>{{ page.title | escape }} | Ethereum Improvement Proposals</title>
+    <meta
+      property="og:title"
+      content="{{ page.title | escape }} | Ethereum Improvement Proposals"
+    />
+    <meta name="description" content="Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards." />
+    <meta property="og:description" content="Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards." />
+    <meta name="twitter:description" content="Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards." />
+  {% endif %}
+  <meta name="generator" content="Jekyll v3.8.5" />
+  <meta property="og:locale" content="en_US" />
+  <link rel="canonical" href="https://eips.ethereum.org{{page.url}}" />
+  <meta property="og:url" content="https://eips.ethereum.org{{page.url}}" />
+  <meta property="og:site_name" content="Ethereum Improvement Proposals" />
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:site" content="@ethereum" />
+  <script type="application/ld+json">
+    {
+      "@type": "WebSite",
+      "url": "https://eips.ethereum.org/",
+      "name": "Ethereum Improvement Proposals",
+      "description": "Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.",
```

Comment:
Is it possible to get these fields and the URL from `_config.yml` ?

Reply:
I actually made this change myself, hope you don't mind.

Reply:
Makes sense to me!